### PR TITLE
Feat task layout

### LINKED ISSUE #2: Create TODO task view in sample application
Create TODO task view in the sample application.
Please add more info here about layout and technology that it will be used.

### PR DESCRIPTION
**Pull request by Lavanyagaur22**
**Description** 

Create TODO task view in the sample application. ( fix : #2 )

**Details**

1. To add a new task - **Floating Action Button** is used.
2. Click on fab -> a custom **alert dialog** is popped up which takes in the title and the detail of the task.
3. The task is then displayed in a **recycler view**.

Image of the sample app:- 

[](https://postimg.cc/Zv3F4YMw)

### CODE REVIEW

**Comment by wtrocki** [comment]
The layout seems to be using colors of the palette. Is that intentional?
I would use standardized material design colors: 
https://material.io/tools/color/#!/?view.left=0&view.right=0

Pick your fav base and simply drop new style. 
Inside views always rely on style instead of implicit colors.

File: .idea/libraries/Gradle__android_arch_lifecycle_livedata_1_1_1.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<component name="libraryTable">
```
Comment (wtrocki) [inline .idea/libraries/Gradle__android_arch_lifecycle_livedata_1_1_1.xml:1]: Add gitignore for those files. https://github.com/github/gitignore/blob/master/Android.gitignore

**Review by wtrocki — CHANGES_REQUESTED** [review]
Let's remove .idea files.

File: .idea/libraries/Gradle__android_arch_lifecycle_livedata_1_1_1.xml (hunk 1 above)
Comment (Lavanyagaur22) [reply .idea/libraries/Gradle__android_arch_lifecycle_livedata_1_1_1.xml:1]: Okay.

**Comment by wtrocki** [comment]
.idea folder is still pushed in this PR. IF we remove it and add those changes to git then it will disappear.

**Review by wtrocki — APPROVED** [review]
Going to clean things later.

**Pull request merged**